data-esnext.js

```diff
@@ -51,7 +51,7 @@ exports.tests = [
   spec: 'http://wiki.ecmascript.org/doku.php?id=strawman:do_expressions',
   exec: function () {/*
     return do {
-      let x = 23;
+      var x = 23;
```

Comment:
I've just noticed that it might test something it's not supposed to, and sent PR. Your call